### DISCUSSION TITLE
Mouse Coursor randomly stops

### DISCUSSION THREAD

**Discussion by lisekilis** [ppy/osu#27715] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Seperate from https://github.com/ppy/osu/issues/27227 because It reffers to tablet and I'm using mouse
The mouse cursor randomly stops on jumps in lazer since the the lastest patch (didn't happen before and doesn't happen in stable)
Mouse used is the Corsair M65 1200dpi @ 1000hz (anything above 1000hz breakes the game, will make a seperate issue)
Ingame settings:
High Precision mouse @ 0.8 sensitivity, Fullscreen, multithreaded Dx11 Frame limit: 8x @ 1080p 240hz 
PC: 
![image](https://github.com/ppy/osu/assets/101191226/ba83035b-6e5d-4946-9809-7559168616a7)
(if that's relevant) 

### Screenshots or videos

[Replay](https://github.com/ppy/osu/files/14735410/lisekilis.playing.sad.alex.-.new.heartbreak.Coolrrrray.heartless.2024-03-24_13-08.zip)
the sharp cutoff happens at: 
1. 1:15
2. 1:37
3. 2.11
video:
https://github.com/ppy/osu/assets/101191226/17d4b3f0-ea90-4651-a0d6-c8342eab4c4e

### Version

2024.312.1-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14735595/compressed-logs.zip)

**Comment by Stoppedpuma** [comment]
> anything above 1000hz breakes the game, will make a seperate issue

Can you explain what "breaks" the game? This seems to be more likely an issue on your end since I haven't seen any other posts about this and I can't reproduce any issues. Keep in mind **osu! will not use more than 1000hz** so using a value higher than 1000 pointless.

If you're able to reproduce this consistently can you capture a video with the graphs shown when you press Ctrl + F11 twice?

**Reply by lisekilis** [reply, marked as answer]
Sorry for the late response
I have a discussion on this subject, you can find it here https://github.com/ppy/osu/discussions/27718
